lib/CPANRatings/Control.pm

```diff
@@ -62,6 +64,60 @@ sub _calc_auth_token {
     return '1-' . sha1_hex('8wae4ko -  this is very secret!' . $cookie);
 }
 
+sub as_json {
+    my ($self, $reviews, $mode, $id) = @_;
+
+    my $data = {
+        $mode => $id,
+        reviews => [],
+    };
+
+    # arbitrary choice of one year being outdated...
+    my $outdated = DateTime->now->subtract( years => 1 );
```

Comment:
a slightly fancier version would be "a year ago or at the most recent release; whichever is later" or something like that.